src/uvwasi.c

```diff
@@ -10,7 +10,6 @@
 # define SLASH_STR "/"
 # define IS_SLASH(c) ((c) == '/')
 #else
-# include <processthreadsapi.h>
 # define SLASH '\\'
```

Comment:
This won't be needed. It will cause unknown target error on visual studio compiler.

Reply:
`SwitchToThread()` is used in `uvwasi_sched_yield()`. According to https://docs.microsoft.com/en-us/windows/win32/api/processthreadsapi/nf-processthreadsapi-switchtothread, we need to include processthreadsapi.h. If we don't need that, should we at least be including windows.h?

Reply:
Looks won't work.
```console
C:\Program Files (x86)\Windows Kits\10\include\10.0.17763.0\um\winsock.h(303): note: see previous definition of 'INADDR_BROADCAST'
C:\Program Files (x86)\Windows Kits\10\include\10.0.17763.0\shared\ws2def.h(633): error C2011: 'sockaddr_in': 'struct' type redefinition
C:\Program Files (x86)\Windows Kits\10\include\10.0.17763.0\um\winsock.h(309): note: see declaration of 'sockaddr_in'
C:\Program Files (x86)\Windows Kits\10\include\10.0.17763.0\um\winsock2.h(136): error C2011: 'fd_set': 'struct' type redefinition
C:\Program Files (x86)\Windows Kits\10\include\10.0.17763.0\um\winsock.h(65): note: see declaration of 'fd_set'
```

Reply:
My suspect is that `libuv.h` already include this, so this won't be necessary ?